README.md

```diff
@@ -3,14 +3,48 @@
 This is a library to help with authenticating requests to and from PeekPro when
 writing apps.
 
-## Config:
+## Configuration
+
+### Basic Configuration
 
 ```elixir
 config :peek_app_sdk,
   peek_app_secret: "APP_SECRET",
-  peek_app_id: "APP_ID"
+  peek_app_id: "APP_ID",
+  peek_api_url: "https://api.peek.com",
+  peek_app_key: "APP_KEY"
+```
+
+### Multiple Application Configurations
+
+PeekAppSDK supports multiple configurations, allowing different applications to use their own credentials:
+
+```elixir
+# Default configuration
+config :peek_app_sdk,
+  peek_app_secret: "DEFAULT_SECRET",
+  peek_app_id: "DEFAULT_APP_ID",
+  peek_api_url: "https://api.peek.com",
+  peek_app_key: "DEFAULT_APP_KEY"
+
+# Application-specific configuration
+config :semnox,
+  peek_app_secret: "SEMNOX_SECRET",
+  peek_app_id: "SEMNOX_APP_ID"
 ```
 
+Then use the application identifier when calling PeekAppSDK functions:
+
+```elixir
+# Using default configuration
+PeekAppSDK.query_peek_pro("install_id", "query { test }")
+
+# Using application-specific configuration
+PeekAppSDK.query_peek_pro("install_id", "query { test }", %{}, :semnox)
```

Comment:
So the only way we can use multi-config is if we also do an umbrella app? That feels like a shame. I wonder if we can have the SDK not care about this and have that be up to the parent app (umbrella or non-umbrella). Like maybe we can have something like

```
config :peek_app_sdk, apps: [semnox: [id: "fo", secret": "bar"]]
```
And then it can lookup the app with the key you provide.

That way the umbrella app can do what you're currently doing but if we want to not use an umbrella, things still work?